UPGRADE-2.0.md

```diff
@@ -0,0 +1,105 @@
+UPGRADE FROM 1.x to 2.0
```

Comment:
Maybe add a TLDR? Something like: "Many internal things in the bundle has changed. If you only instlaled the bundle and added some configuration then there is super simple to upgrade. "